curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a3a488b24fb32b91155d56.md

```diff
@@ -10,7 +10,7 @@ dashedName: step-7
 Variables can be <dfn>assigned</dfn> a value. When you do this while you declare it, this is called <dfn>initialization</dfn>. For example:
 
 ```js
-let camperbot = "Bot";
+let age = 32;
 ```
 
 This would initialize the `camperbot` variable with a value of `Bot`, a string.
```

Comment:
This needs to change too.
Re read through the issue again to make sure all of the changes are met

https://github.com/freeCodeCamp/freeCodeCamp/issues/53424

Reply:
Sorry, i missed that